src/language/grammar.tx

```diff
@@ -1,5 +1,6 @@
 Specification:
 	actors += Actor*
+	( 'maneuverJunction' '(' maneuverJunction = ID ')' )?
```

Comment:
I assume there would be at most one maneuverJunction?

Reply:
Okay, so you went with the approach that the maneuverjunction is given as a clause and ever "doesmaneuver" is applied to that, without needing to refer to the maneuverJunction. Is this so?

In that case, one maneuverJunction is enough.

Tho we should think about whether the maneuverJunctions should be formatted as a constraint (as is done currently) or as a object (e.g. "JunctionUnderTest j1234")

Reply:
> Okay, so you went with the approach that the maneuverjunction is given as a clause and ever "doesmaneuver" is applied to that, without needing to refer to the maneuverJunction. Is this so?
> 
> In that case, one maneuverJunction is enough.

Oh, not really. In the current grammar, there is a region in `doesManeuver` constraints. So, is it not really necessary to have both `maneuverJunction` and a region in `doesManeuver`?

> Tho we should think about whether the maneuverJunctions should be formatted as a constraint (as is done currently) or as a object (e.g. "JunctionUnderTest j1234")

Sure. At this time, it just looks like a constraint, but is actually a single attribute in `Specification`.